web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -414,11 +443,11 @@ const MemberSelectorDialog = ({ open, selectedMembers, onClose, onSubmit }) => {
                   {...params}
                   variant="outlined"
                   label="Filter Members"
-                  placeholder={`Search for ${filterType.toLowerCase()}`}
+                  placeholder={`Search for ${filterType?.toLowerCase()}`}
```

Comment:
I suppose `filterType` is always set, but you had to add the `?` to make tooling happy. Right?

Reply:
This was unnecessary. Removed. Good eyes!